lms/static/js/spec/edxnotes/plugins/events_spec.js

```diff
@@ -84,6 +97,18 @@ define([
             expect(this.annotator.plugins.Events.oldNoteText).toBeNull();
         });
 
+        it('should not log the edx.course.student_notes.edited event if the note is new', function() {
+            var old_note = noteWithoutId,
+                new_note = $.extend({}, noteWithoutId, {text: 'text-456'});
+
```

Comment:
@polesye Please use CamelCase notation for JS variables: `old_note` --> `oldNote` ; `new_note` --> `newNote` throughout this file.